### PR TITLE
feat: blog and launch post

### PR DESCRIPTION
**Pull request by alanshaw**
The blog was pretty simplistic so I added the `@vuepress/plugin-blog` plugin which gives us the following features:

* Friendly URLs
* RSS feeds
* Automatically generated index page
* Post categorization with tags
* Pagination

I've put the social card in as a placeholder graphic. Max width is 740px so a 1480px wide graphic will look crisp on HD screens. The social card is 388px high (776px @ 2x), but there's no constraint on that.

I've done this all as quickly as possible so there's some code duplication that needs cleaning up and there's no renderer for the tags so it currently just shows a blank page.

Index page:

Screenshot 2020-08-03 at 14 11 45

Post page:

Screenshot 2020-08-07 at 14 38 52

### CODE REVIEW

**Comment by johnnymatthews** [comment]
@alanshaw, sorry I approved your changes earlier, but didn't actually get a chance to run through the initial post. And when I came round to doing it now I couldn't request changes. So I've just made the changes and pushed directly into this branch. I hope that's ok. Feel free to revert anything that doesn't look right to you.

Also, [line 100 in the initial post](https://github.com/drand/website/pull/67/files#diff-468a4a186e960ec301697591c67b4c60R100) call for a link to the _logo assets folder_. Not sure where you mean, maybe [`/docs/.vuepress/public/images`](https://github.com/drand/website/tree/master/docs/.vuepress/public/images)?

File: docs/blog/2020-08-10-drand-launches-v1.0.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+With this new governance model, we've established the rules and requirements that the League of Entropy needs in order to maintain a high level of network security, ensure that the League operates effectively, and establish the right conditions for ongoing improvement to the production quality of drand. We've also laid down the procedures for adding new members to the League of Entropy in an equitable, fair, and decentralized way. All told, we think these changes set up the drand project for continued success as we find new users and implement new features.
+
+## New website & logo
+
+We built this website to be the drand community's primary repository for the growing body of documentation about drand. We also want this to be the new home for the drand community of developers, operators, and end-users. We'll use it to host our blog and share important updates and news about the project.
+
+In addition to a new site, we also created a logo for the drand project, which you can see at the top of this post or [here](LINK TO LOGO ASSETS FOLDER) for a logo kit.
```
Comment (johnnymatthews) [inline docs/blog/2020-08-10-drand-launches-v1.0.md:100]: Not sure where this `LINK TO LOGO ASSETS FOLDER` thingy is.

Comment (jpgross3) [reply docs/blog/2020-08-10-drand-launches-v1.0.md:100]: I sent this to Alan in Slack to house it in the appropriate place: https://docs.google.com/presentation/d/1xQxONjpVU6DCIf7JhcfrFgejfExHe-NR0qNCuB-zaZQ/edit?usp=sharing

File: docs/blog/2020-08-10-drand-launches-v1.0.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+A set of tools is now used to monitor the various drand nodes and alert operators to incidents. A [status page](https://drand.statuspage.io/) that shows both system status and any planned maintenance is now available for the network. We have also established a network observatory to measure network performance and reliability from the perspective of the end-user, with vantage points around the world.
+
+### Drand’s specification & security model
+
+The drand protocol now has a [full specification](/docs/specification/) and a documented [security model](/docs/security-model/). 
+
+### Security audit
+
+The drand implementation underwent an end-to-end security audit by SigmaPrime, and all relevant issues were resolved.
+
```
Comment (jsoares) [inline docs/blog/2020-08-10-drand-launches-v1.0.md:54]:
```suggestion
### Security audit

The drand implementation underwent an end-to-end security audit by SigmaPrime, and all relevant issues were resolved.

```

We have nothing to announce here.

File: docs/blog/2020-08-10-drand-launches-v1.0.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+title: Drand launches v1.0, and the League of Entropy becomes a production service!
+summary: Welcome to the brand new website for the drand project! Drand is now in production, providing publicly verifiable randomness as a foundational internet service.
+date: 2020-08-10
+tags:
+    - Release
+    - Major
+    - League of Entropy
+---
+
+![Placeholder launch image](/images/social-card.png)
```
Comment (jsoares) [inline docs/blog/2020-08-10-drand-launches-v1.0.md:11]: ^ labelled as a placeholder. It seems a little redundant to have a drand logo as a cover image in the drand website. It seems we have the opengraph image set so the social card will be used for SNS previews anyway.

**Comment by jsoares** [comment]
> New title based on Nicolas' suggestion to put LoE first: "**The League of Entropy launches drand v1.0 to become the Internet’s first production-grade, publicly verifiable randomness beacon!**"
> 
> I'm not wedded to it, but if there is no pushback, let's change it.

@jpgross3 We changed the Pinkston spec sheet to mention drand v1.1.1 instead. Either is fine (one is the big milestone, the other is the current version) but we should be consistent. My suggestion would be to use v1.1.1 but remove the version from the title (because it looks weird to announce a point release). We'd therefore be announcing _becoming a production service_ and then, in the text, we mention the release v1.1.1.

File: docs/blog/2020-08-10-drand-launches-v1.0.md (hunk 3 above)
Comment (alanshaw) [reply docs/blog/2020-08-10-drand-launches-v1.0.md:11]: It is a placeholder until this issue is resolved: https://github.com/protocol/design-shop/issues/344

Comment (jsoares) [reply docs/blog/2020-08-10-drand-launches-v1.0.md:11]: Missed that one, thanks!

**Comment by alanshaw** [comment]
> We'd therefore be announcing _becoming a production service_ and then, in the text, we mention the release v1.1.1.

@jsoares I think that's fine. I'd originally commented on this in the google doc:

> IMHO the big news is the major version number, not the feature or bugfix releases that followed. We're trying to highlight that drand is now stable and production ready, not that we released a bugfix.
>
> So to me, it doesn't matter that v1.0 is not the latest version since the point is to highlight that we transitioned from a non-stable v0.x version. As a side point, yeah, we also released a feature and a bugfix since then but it's not the big news we're trying to highlight here. By including the full latest version number we risk people thinking this is just a run-of-the-mill bugfix post and glossing over it.

**Comment by jpgross3** [comment]
Just to make sure we all agree. Currently:

- Press release makes no mention of the specific version
- Specs sheet has no version in the title, mentions v1.1.1 in the Q&A
- Drand.love blog post (this one) announces v1.0 in the title and then lists the new features of v1.0.0

It sounds like we want uniformity, but also consider v1.0 the BIG announcement. With that said, how does this sound:

- Press release stays the same
- Specs sheet discusses v1.1
- Current blog post changes the text to: 

Title: 
# The League of Entropy launches drand v1.0 to become the Internet’s first production-grade, publicly verifiable randomness beacon!

Subsection in blog: 
## Drand v1.1
Drand reached v1.0 in July and v1.1 in August, rolling out many new features and architecture improvements...

**Comment by jpgross3** [comment]
**(In general, my understanding is that we should drop the final .1 or .0, regardless.)**

**Comment by jsoares** [comment]
Sounds good to me. While v1.0 is the but milestone. some of the improvements listed (chain following, broadcast channel) are indeed 1.1 features.

File: docs/blog/2020-08-10-drand-launches-v1.0.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+We believe that drand can be a foundational Internet core service for randomness, one that everyone online can access and use in the same way as other foundational protocols, like DNS or NTP.
+
+## Drand v1.1
+
+Drand launched v1.0 in July and updgraded to v1.1 in August, rolling out many new features and architecture improvements. Chief among them are:
+
+### 3 layered architecture
+
+The network architecture was split into three layers, offering enhanced protection to the drand nodes while increasing scalability, with a distribution network utilizing CDNs, [libp2p Gossipsub](https://blog.ipfs.io/2020-05-20-gossipsub-v1.1/) relays, and [Twitter bots](https://twitter.com/loebot).
```
Comment (jsoares) [inline docs/blog/2020-08-10-drand-launches-v1.0.md:33]:
```suggestion
The network architecture was split into three layers, offering enhanced protection to the drand nodes while increasing scalability, with a distribution network utilizing [HTTP CDNs](http://api.drand.sh/public/latest), [libp2p Gossipsub](https://blog.ipfs.io/2020-05-20-gossipsub-v1.1/) relays, a [Tor hidden service](http://drandeokfd5aaz3hr4hfk7mlr23vc63boxrpr44ertumtbousmdgbhad.onion/public/latest), and a [Twitter bot](https://twitter.com/loebot).
```

File: docs/blog/2020-08-10-drand-launches-v1.0.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+A set of tools is now used to monitor the various drand nodes and alert operators to incidents. A [status page](https://drand.statuspage.io/) that shows both system status and any planned maintenance is now available for the network. We have also established a network observatory to measure network performance and reliability from the perspective of the end-user, with vantage points around the world.
+
+### Drand’s specification & security model
+
+The drand protocol now has a [full specification](/docs/specification/) and a documented [security model](/docs/security-model/). 
+
+### Security audit
+
+The drand implementation underwent an end-to-end security audit carried out by SigmaPrime. We are excited to share the report with you [here](https://drive.google.com/file/d/1fCy1ynO78gJLCNbqBruzHx7bh72Tu-q2/view?usp=sharing) in which you can learn about the findings and the approach used. Working with the SigmaPrime team was absolute delight, they were throrough and very communicative throughout the entire audit, which enabled the drand team to quickly solve the issues as they were found.
```
Comment (alanshaw) [inline docs/blog/2020-08-10-drand-launches-v1.0.md:53]:
```suggestion
The drand implementation underwent an end-to-end security audit carried out by [SigmaPrime](https://sigmaprime.io/). We are excited to share the report with you [here](https://drive.google.com/file/d/1fCy1ynO78gJLCNbqBruzHx7bh72Tu-q2/view?usp=sharing) in which you can learn about the findings and the approach used. Working with the SigmaPrime team was an absolute delight, they were thorough and very communicative throughout the entire audit, which enabled the drand team to quickly solve the issues as they were found.
```

**Pull request merged**